### PR TITLE
PPTP-1433 | content updates start page

### PR DESCRIPTION
**Pull request by liampan**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [ ] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtax/registration/views/StartViewSpec.scala [hunk 1]
```diff
@@ -124,7 +107,7 @@ class StartViewSpec extends UnitViewSpec with Matchers {
 
       view.getElementsByClass("govuk-button").first() must containMessage("startPage.buttonName")
       view.getElementsByClass("govuk-button").first() must haveHref(
-        uk.gov.hmrc.plasticpackagingtax.registration.controllers.routes.StartRegistrationController.startRegistration().url
+        uk.gov.hmrc.plasticpackagingtax.registration.controllers.routes.StartRegistrationController.startRegistration()
```
[the remark this answers is not in the document]
Comment (liampan) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/views/StartViewSpec.scala:110]: I had some failures with this `Element attribute {href} had value {/start-registration}, expected {/register-for-plastic-packaging-tax/start-registration}` even before I changed it, I can only think it was something to do with having it running locally at the same time, as it passed no problems in jenkins. Have you seen anything like this before?

[the remark this answers is not in the document]
Comment (RyanKellyHMRC) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/views/StartViewSpec.scala:110]: I frequently see failures like that when running tests in isolation: either in IJ or via sbt (testQuick/testOnly). Haven't got to the bottom of it yet.

**Review by RyanKellyHMRC — APPROVED** [review]
LGTM

**Comment by liampan** [comment]
Screenshot 2022-02-11 at 10 00 44

**Pull request merged**